src/Compilers/Core/Portable/Compilation/SemanticModel.cs

```diff
@@ -63,6 +63,14 @@ public SyntaxTree SyntaxTree
         protected abstract SyntaxTree SyntaxTreeCore { get; }
 
         /// <summary>
+        /// Returns true if this is a SemanticModel that ignores accessibility rules when answering semantic questions.
+        /// </summary>
+        public virtual bool HasAccessChecksSuppressed
```

Comment:
Can we name this `SuppressAccessChecks`?  

The pattern of `SuppressY` is used a number of times in our code base both internally and publically.  The pattern of `HasYSuppressed` does not appear anywhere I could find though we do have a few occurrences of `IsYSuppressed` but they are all methods and internal.  

That pattern seems to be more consistent with other public, and even internal, APIs that we expose.  


@gafter what do you think?